highlighter/__init__.py

```diff
@@ -43,7 +43,7 @@ def highlight_text(text, expr):
         @:return marked text, e.g., "sample text <mark>highlighted part</mark> rest of the text"."""
         result = text
 
-        # TODO: add an implementation
+        result = text.replace(expr, "<mark>" + expr + "</mark>")
```

Comment:
Your implementation is case sensitive, am I wrong?

Reply:
Yes you are right 
Not sure where this requirement comes from to be case insensitive ?

Reply:
There is no such a requirement but without it this task is too easy 😀